src/lib.rs

```diff
@@ -185,6 +187,13 @@ impl<T> Glog<T> {
         }
     }
 
+    pub fn with_short_file_names(self, with_short_file_names: bool) -> Glog<T> {
```

Comment:
I understand that this behavior is meant to match `glog`'s, but I can see "short" being somewhat ambiguous (e.g., what qualifies as "short"? A single directory preceding it?).

How would you feel about renaming this to `with_trimmed_directory` and maybe adding a seperate builder option that exposes [`Path::strip_prefix`](https://doc.rust-lang.org/stable/std/path/struct.Path.html#method.strip_prefix), allowing for more fine-grained rendering of paths?

Reply:
You're right, I'll use Path and rename.

Not sure about using `Path::strip_prefix`. Could be useful, though with the module shown (`with_target(true)`) we would never use it. Either the full path or just the filename, I doubt that stripping away just a part makes much sense.

Reply:
For Rust there's a very high likelihood that the source will be called `lib.rs` (or maybe `mod.rs`) and even with one directory element you'll get `src/lib.rs`. Are you sure that this change will actually give you useful logs?

Reply:
In combination with `with_target(true)`, otherwise not.